### PR TITLE
Scheduler views

### PR DESCRIPTION
**Pull request by CalvinFronda**
Changed styles and code review issues.

### CODE REVIEW

File: omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js [hunk 1]
```diff
@@ -64,72 +65,160 @@ class SessionView extends Component {
...
+                instructor: instructor
             }
         })
+    }
 
 
+
+    // Formatting the date in the view
+    formatDate = (start, end) => {
+        let DayConverter = {
```
Comment (NeilAllavarpu) [inline omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js]: Constant

File: omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js [hunk 2]
```diff
@@ -64,72 +65,160 @@ class SessionView extends Component {
...
+            1: "Monday",
+            2: "Tuesday",
+            3: "Wednesday",
+            4: "Thursday",
+            5: "Friday",
+            6: "Saturday",
+        };
+        let MonthConverter = {
```
Comment (NeilAllavarpu) [inline omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js]: Constant

File: omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js [hunk 3]
```diff
@@ -64,72 +65,160 @@ class SessionView extends Component {
...
+        //Gets months
+        let Month = MonthConverter[startMonth]
+
+        //Start times and end times variable 
+        let startTime = start.slice(11)
+        let endTime = end.slice(11)
+
+        // Converts 24hr to 12 hr time 
+        this.timeConverter = (time) => {
+            let timeString = time;
```
Comment (NeilAllavarpu) [inline omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js]: Useless assignment, you can just declare this on line 130 (you assign it a value without ever using the initialized value here).

File: omou-frontend/src/components/FeatureViews/Scheduler/Scheduler.js [hunk 4]
```diff
@@ -201,10 +209,12 @@ class Scheduler extends Component {
                 session["description"] = this.props.courses[session.course_id].description;
                 session['type'] = this.props.courses[session.course_id].type;
                 session['resourceId'] = this.props.courses[session.course_id].room_id;
+                session['room_id'] = this.props.courses[session.course_id].room_id;
```
[the remark this answers is not in the document]
Comment (CalvinFronda) [reply omou-frontend/src/components/FeatureViews/Scheduler/Scheduler.js]: sounds good

**Pull request closed without merging**